src/dra/692E8.c

```diff
@@ -801,15 +801,15 @@ void func_8010FB24(void) {
     func_8010E3E0();
 }
 
-void func_8010FB68(void) { // Related to Dark Methamorphosis
+void func_8010FB68(void) { // Related to Dark Metamorphosis
     PLAYER.velocityY = 0;
     PLAYER.velocityX = 0;
     SetPlayerStep(Player_SpellDarkMetamorphosis);
     func_8010E3E0();
     func_8010DA48(0xBA);
     PlaySfx(NA_SE_VO_AL_DARK_METAMORPHOSIS);
     PlaySfx(NA_SE_PL_MP_GAUGE);
-    g_Player.D_80072F16 = func_800FDB18(3, 0x400);
+    g_Player.D_80072F16 = GetStatusAlimentTimer(3, 0x400);
```

Comment:
`STATUS_ALIMENT_DARK_METAMORPHOSIS, 0x400`? Feel free to go ahead and merge if you disagree, the PR is good either way.

Reply:
Oh actually ALIMENT -> AILMENT

Reply:
Good catch :P I will fix it in a bit as I am in the middle of another branch